test-utils/src/dev/db.rs

```diff
@@ -1274,6 +1281,9 @@ mod test {
             client2.query("SELECT v FROM foo", &[]).await.expect("list rows");
         assert_eq!(rows.len(), 0);
         client2.cleanup().await.expect("second connection closed ungracefully");
+
+        db1.cleanup().await.expect("failed to clean up first database");
```

Comment:
These tests always should have been cleaning up the database instead of relying on the best-effort `Drop` impl.  Now that the `Drop` impl doesn't clean these up any more, they were leaking the directory without this fix.